src/methods/geom_relations/common.jl

```diff
@@ -0,0 +1,28 @@
+# Identical boilerplate methods for geom relations live here
+
+for f in (:coveredby, :crosses, :disjoint, :overlaps, :touches, :within)
+    _f = Symbol(:_, f)
+
+    # Features
+    @eval begin
+        $_f(::FeatureTrait, g1, ::Any, g2) = $f(geometry(g1), g2)
+        $_f(::Any, g1, ::FeatureTrait, g2) = $f(g1, geometry(g2))
+        $_f(::FeatureTrait, g1, ::FeatureTrait, g2) = $f(geometry(g1), geometry(g2))
```

Comment:
Let's forward args... and kwargs... here?  There is T = Float64 and all the kwargs like exact

Reply:
Ah true, maybe I got confused with the clipping which does have that